pkg/networkmanager/index.html

```diff
@@ -156,7 +156,7 @@ <h2 class="panel-title" translatable="yes">Unmanaged Interfaces</h2>
 
   <script id="network-vlan-settings-template" type="x-template/mustache">
     <form class="ct-form">
-      <label class="control-label" translatable="yes">Parent</label>
+      <label class="control-label" for="network-vlan-settings-parent-select" translatable="yes">Parent</label>
```

Comment:
The [spec](https://developer.mozilla.org/de/docs/Web/HTML/Element/label) says "The id of a labelable form-related element", and `<div>` is not [labelable](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Form_labelable) -- only <input>s are. So I wonder if this will actually work as intended. Ideally the dynamically generated dropdown-select would have an id that this should link to. Or perhaps  move the <label> itself into the auto-generation, so that id= and for= can be generated in the same place?

Same for all the others in this file.